### PR TITLE
weatherdata quickstart

### LINKED ISSUE #46: Weather Data Stack
Extract recent weather data from open APIs (Weatherstack), load it into a data warehouse using Airbyte, and analyze weather patterns or predictions.

How to get started: 

- Check out the [README](https://github.com/airbytehq/quickstarts/tree/main) and the [Contribution instructions](https://github.com/airbytehq/quickstarts/blob/main/CONTRIBUTING.md) in this repo.
- Check one of the [existing quickstarts](https://github.com/airbytehq/quickstarts#list-of-available-quickstarts) and observe their structure.
- (Optional, but highly recommended) Implement one of the quickstarts by following the instructions in their README. I recommend starting with [E-commerce analytics in BigQuery](https://github.com/airbytehq/quickstarts/tree/main/ecommerce_analytics_bigquery)

### PR DESCRIPTION
**Pull request by Ronlin1**
Closes #46

### CODE REVIEW

**Comment by Ronlin1** [comment]
Please hold up! I am fixing some issues with the API

**Comment by bishalbera** [comment]
@ThaliaBarrera I believe he is encountering issues related to Terraform. As per the [documentation](https://registry.terraform.io/providers/airbytehq/airbyte/latest/docs), Weatherstack is not currently supported through Terraform. I have attached the error picture below for reference. Additionally, you cannot access historical and forecast data in the free tier of the Weatherstack API, which is another problem. 😖 

image

**Solution**
- The current solution is to use the Airbyte UI as of now.

**Comment by Ronlin1** [comment]
I believe we are good to go @ThaliaBarrera

**Comment by Ronlin1** [comment]
I believe we are good to go @ThaliaBarrera

File: weather_data_stack/README.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
+# Weather Data Stack with Airbyte, dbt, Dagster and BigQuery
+
+Welcome to the "Weather Data Stack" repository! 🌟 This is your go-to place to easily set up a data stack using Airbyte, dbt, BigQuery, and Dagster. With this setup, you can pull weather data from WeatherStack API, put it into BigQuery, and play around with it using dbt and Dagster.
+
+This Quickstart is all about making things easy, getting you started quickly and showing you how smoothly all these tools can work together!
+
+Here is a visual representation of how data flows through our integrated tools in this Quickstart.
+
```
Comment (ThaliaBarrera) [inline weather_data_stack/README.md:8]: @Ronlin1 Could you include the architecture diagram here?

Comment (ThaliaBarrera) [reply weather_data_stack/README.md:8]: Oh! I see it's below in the architecture diagram section. Maybe then just removing this sentence not to be confusing.

File: weather_data_stack/infra/airbyte/main.tf [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+
+    query = var.query
+
+    historical_date = "2023-09-28"
+
+  }
+  name         = "Weatherstack"
+  workspace_id = var.workspace_id
+}
```
Comment (ThaliaBarrera) [inline weather_data_stack/infra/airbyte/main.tf:22]: @Ronlin1 does this actually work? I don't see the `airbyte_source_weatherstack` listed in the terraform documentation. If it doesn't work you can just use the Airbyte UI to create the connectors, but in that case, please remove all the references to Terraform from the quickstart

Comment (Ronlin1) [reply weather_data_stack/infra/airbyte/main.tf:22]: No, as @bishalbera said previously yes its not listed in the terraform docs. I only used the airbyte ui to create the connectors . so should I just delete the infra/airbyte file entirely as it involves terraform

Comment (Ronlin1) [reply weather_data_stack/infra/airbyte/main.tf]: Verify, if am good to go

Comment (Ronlin1) [reply weather_data_stack/infra/airbyte/main.tf]: Hi @ThaliaBarrera , waiting on you.

Comment (ThaliaBarrera) [reply weather_data_stack/infra/airbyte/main.tf]: The PR is merged @Ronlin1!

**Pull request merged**